runtime/rasdump/javadump.cpp

```diff
@@ -520,6 +523,9 @@ JavaCoreDumpWriter::JavaCoreDumpWriter(
 	if (_Error) {
 		j9nls_printf(PORTLIB, J9NLS_ERROR | J9NLS_STDERR, J9NLS_DMP_ERROR_IN_DUMP_STR, "Java", _FileName);
 		Trc_dump_reportDumpError_Event2("Java", _FileName);
+	} else if (_IsSTDOUTERR) {
+		j9nls_printf(PORTLIB, J9NLS_INFO | J9NLS_STDERR, J9NLS_DMP_WRITTEN_DUMP_STDOUTERR_STR, "Java");
```

Comment:
This should be written to stdout if the report was produced on stdout.

Reply:
I think this need to continue to be written to stderr. All the NLS error codes and messages have always gone to stderr, I don't think it should be changed for one particular message. For example, `-Xdump:java:events=vmstart` produces the following messages.
```
JVMDUMP039I Processing dump event "vmstart", detail "" at 2019/02/14 13:45:20 - please wait.
JVMDUMP032I JVM requested Java dump using 'jdk-11.0.2+8/javacore.20190214.134520.3456.0001.txt' in response to an event
JVMDUMP010I Java dump written to jdk-11.0.2+8/javacore.20190214.134520.3456.0001.txt
JVMDUMP013I Processed dump event "vmstart", detail "".
```

Reply:
In light of this, I think we should stick with the existing message (`_FileName` will be `/STDOUT/` or `/STDERR/`).
I think this also means we don't need the new trace event nor the `isSTDOUTERR()` method.